graylog2-server/src/main/java/org/graylog2/plugin/configuration/Configuration.java

```diff
@@ -138,6 +138,10 @@ public void setBoolean(String key, boolean value) {
         return firstNonNull(lists.get(key), defaultValue);
     }
 
+    public boolean listIsSet(String key) {
+        return !lists.isEmpty();
```

Comment:
This needs to check if the list for the given `key` is set. The current code is checking if the map of lists is empty or not. So this needs to be something along the lines of:

```suggestion
        return lists.containsKey(key);
```